survivor.wdl

```diff
@@ -0,0 +1,59 @@
+version 1.0
+
+import "common.wdl"
+
+task Merge {
+    input{
+        Array[File] filePaths
+        Int breakpointDistance = 1000
+        Int suppVecs = 2
+        Int svType = 1
+        Int strandType = 1
+        Int distanceBySvSize = 0
+        Int minSize = 30
+        String sample
+        String outputPath
+
+        String memory = "128G"
```

Comment:
Does it really need this much memory?

Reply:
I can't remember why I put it like that. I think It was when I was running WGS familial melanoma data.